src/lxc/criu.c

```diff
@@ -256,6 +260,10 @@ static void exec_criu(struct criu_opts *opts)
 		if (ret < 0 || ret >= sizeof(log))
 			goto err;
 
+		if (criu_version >= CRIU_IN_FLIGHT_SUPPORT) {
+			DECLARE_ARG("--skip-in-flight");
```

Comment:
On Tue, Jul 05, 2016 at 07:19:17AM -0700, Adrian Reber wrote:
> > @@ -256,6 +260,10 @@ static void exec_criu(struct criu_opts *opts)
> >  		if (ret < 0 || ret >= sizeof(log))
> >  			goto err;
> >  
> > +		if (criu_version >= CRIU_IN_FLIGHT_SUPPORT) {
> > +			DECLARE_ARG("--skip-in-flight");
> 
> I guess adding it to migrate_opts also means that it will be a command-line option to lxc-checkpoint, right?

If you like; we haven't added e.g. predump_dir, or the pageserver
addresses or preserves_inodes flags to lxc-checkpoint, mostly because
I think everyone is using liblxc via the API to migrate stuff, so
that's really the only place we need it.

> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/lxc/lxc/pull/1072/files/ff5fee14f4cae14e1ef49b69a889a3ed81e89bab#r69570663